src/provisioner/provisioner.py

```diff
@@ -144,23 +144,11 @@ def config_apply_cluster(cortx_config_store):
                 f'Error occurred while applying cluster_config {e}')
 
     @staticmethod
-    def cluster_bootstrap(cortx_conf_url: str = None):
-        """
-        Description:
-        Configures Cluster Components
-        1. Reads Cortx Config and obtain cluster components
-        2. Invoke Mini Provisioners of cluster components
-        Paramaters:
-        [IN] CORTX Config URL
-        """
-
-        if cortx_conf_url is None:
-            cortx_conf_url = CortxProvisioner._cortx_conf_url
-        cortx_config_store = ConfigStore(cortx_conf_url)
-
+    def _get_node_info(cortx_config_store: ConfigStore):
```

Comment:
Use small variable name e.g. "cortx_conf" instead of "cortx_config_store". Its in reality cortx_conf only.

Reply:
done.